src/Mailgun/Api/Domain.php

```diff
@@ -31,9 +35,9 @@ class Domain extends AbstractApi
      * @param int $limit
      * @param int $skip
      *
-     * @return DomainListResponse
+     * @return IndexResponse
      */
-    public function listAll($limit = 100, $skip = 0)
+    public function index($limit = 100, $skip = 0)
```

Comment:
I did this because we should reflect REST. We have index, create, show, delete etc